lib/chef/provider/package/powershell.rb

```diff
@@ -17,13 +17,13 @@
 
 require_relative "../package"
```

Comment:
Can we make sure we have a kitchen test that exercises this code change? We may already, but a lot of our resources missing from those tests.